psydac/ddm/petsc.py

```diff
@@ -0,0 +1,112 @@
+# coding: utf-8
+
+import numpy as np
+from itertools import product
+
+from .cart import CartDecomposition
+
+#===============================================================================
+class PetscCart:
+
+    def __init__(self, cart):
```

Comment:
Can you please add a docstring to this nice class?

As usual this should contain the `Parameters` section, and a list of the properties under the `Attributes` section (https://numpydoc.readthedocs.io/en/latest/format.html#class-docstring).

Reply:
Let's leave this to issue #79. Things should be properly documented there.